### PR TITLE
os/arch: Fix Kconfig syntax error at FS_TMPFS_HEAP_INDEX config

### PR DESCRIPTION
**Pull request by KimDongEon**
fix invalid operator

### CODE REVIEW

File: os/arch/Kconfig [hunk 1]
```diff
@@ -768,8 +768,7 @@ config FS_TMPFS_HEAP_INDEX
 	int "TMPFS Heap index"
 	default 1
 	depends on FS_TMPFS
-	depends on MM_NHEAPS > 1
-	depends on MM_REGIONS > 1
+	depends on != MM_NHEAPS
```
Comment (sunghan-chang) [inline os/arch/Kconfig]: Is it valid? `!=MM_NHEAPS`?
How about `MM_NHEAPS != 1` like others?

**Comment by sunghan-chang** [comment]
@KimDongEon Let's write a title like below to know where you modify clearly.
```
os/arch: fix Kconfig syntax error at FS_TMPFS_HEAP_INDEX config
```

**Comment by seinfra** [comment]
Target : [5b942ecfecaaed9410c8db3bed1ebe52bcd3f87a] -

File: os/arch/Kconfig (hunk 1 above)
Comment (KimDongEon) [reply os/arch/Kconfig]: sorry, I mistake modify MM_NHEAPS != 1

**Comment by seinfra** [comment]
Target : [5b942ecfecaaed9410c8db3bed1ebe52bcd3f87a] -

**Comment by seinfra** [comment]
Target : [4944375669f157905aa064bf2d3753406c1f9e5f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4944375669f157905aa064bf2d3753406c1f9e5f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8afd4699adbf974d86157d00e67c67454933c5e8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8afd4699adbf974d86157d00e67c67454933c5e8] - Code Rule Check (C++) OK.

**Pull request merged**